arex-storage-config/src/main/java/com/arextest/config/model/dao/config/AppContractCollection.java

```diff
@@ -0,0 +1,24 @@
+package com.arextest.config.model.dao.config;
+
+import com.arextest.config.model.dao.BaseEntity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.FieldNameConstants;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * @author wildeslam.
+ * @create 2024/7/4 10:57
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+@FieldNameConstants
+@Document(collection = "AppContract")
+public class AppContractCollection extends BaseEntity {
```

Comment:
The arex-storage-config package is referenced by arex-api. If the model and logic is not shared, it is recommended to include it in arex-storage-core.